packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -60,6 +61,7 @@ describe('Incoming Payment Service', (): void => {
 
   afterAll(async (): Promise<void> => {
     await appContainer.shutdown()
+    jest.resetAllMocks
```

Comment:
this function isnt being called (missing `()`) but I'm not sure if this is necessary anyways. We have the [tests configured](https://github.com/interledger/rafiki/blob/main/packages/backend/jest.config.js#L22) to [clear mocks ](https://jestjs.io/docs/configuration#clearmocks-boolean)between all tests already.

Reply:
I did find this necessary otherwise the last system time reset was messing up my tests below and giving some weird NaN values to the date function. But when I included the reset it resolved that. I assumed it as because `afterEach` sets the time back to reality ` jest.useRealTimers()` but it doesn't reset the system time after calling `jest.useFakeTimers()
      jest.setSystemTime(incomingPayment.expiresAt)`?